tests/integration/test_dictionaries_ddl/configs/user_default.xml

```diff
@@ -4,6 +4,7 @@
             <allow_databases>
                 <database>default</database>
                 <database>test</database>
+                <database>Log</database>
```

Comment:
why?

Reply:
I didn't know that
```
<allow_database>
    <database>Log</database>
</allow_database>
```
can give `GRANT TABLE ENGINE ON Log`. If so then it's actually a bug.

Anyway we have a better way to grant privileges in `users.xml`:
```
<users>
    <default>
        <grants>
            <query>GRANT TABLE ENGINE ON Log</query>
        </grant>
    </default>
</users>
```

Reply:
Also it's not logical that
```
<user_name>
</username>
```
makes a user who can use all table engines, whileas
```
<user_name>
    <databases>
        test
    </databases>
</username>
```
makes a user who can use no table engines. I think in both cases all table engines should be allowed (anyway using users.xml to create users is an old-fashioned way).